docs/development/review.md

```diff
@@ -75,51 +129,52 @@ kindly,
-already agreed to solve this using XYZ. So I'm going to have to ask you to rework this. Let me know if I can help.
-
-
-A fourth type he sometimes uses is the reaction, which is not actionable, but might help to express emotions such as thanks.
-
-He further suggests to always keep things as clear as possible by using the following pattern:
-
->[type]: [actionable request] 
-
->[rationale or discussion]
-
-Whereas:
-- type: one of Question, Suggestion or Changes Request.
-- actionable request: is a short clear question or statement, which makes clear how the reviewee
-should respond.
-- rational or discussion: Possible further explanations of the reasoning behind the request.
-
-An example could be: 
->Suggestion: use XYZ. 
-
->You may have come across this already, but we might be able to use XYZ for this because blah blah blah etc.
-
-We adapted this slightly by bundling questions, suggestions and change requests in grouped listings and placing
-the reactions up front.
-
-## Inline Comments
-Github makes it very easy to leave inline comments. However, in the past, we observed, that large
-numbers of inline comments make a review hard to read and understand. We therefore propose
-only the leave inline comments for trivial directly actionable suggestions and change request, that
-do not need an rational (E.g. *Change Request: typo in word actionalbe*).
-
-Do not leave any reactions in inline comments. They tend to crowd the discussion without giving
-the possibility of being resolved (seens not actionable). Move them to the summary.
+Please commit the following file as part of the review to the root of the component.
 
+#### If the file already exists:
+* Please update the sections 'Maintenance' and 'Statistics'.
+* Please add the sections 'Criteria based Review for [ilias_version]' und 'Qualitative Review for [ilias_version]' **right below** the section 'Statistics'
 
+#### Review.md
 
+```
+# Review of `[name_of_component]` by [name_of_reviewer] at [date_of_review]
+
+## Maintenance
+* Maintainers/Coordinators:
+* Committers in last 365 days:
+
+## Statistics
+* Lines of Code: 
+* Nr of Tests: 
+* LoCs changed in last 365 days:
+* Nr of Commits in last 365 days: 
+* Nr of Open Mantis Issues: 
+* Mantis Issues resolved in last 365 days: 
+* Nr of Votes in last 365 days:
+* Nr of Votes for currently open issues:  
+
+## Criteria based Review for [ilias_version]
+
+* [ ] `[name_of_component]`and its administration can be used in devmode with PHP 8 and 8.1 without producing PHP-specific errors
+* [ ] The code style is PSR-12 + snake case compliant
+* Globals:
+	* [ ] The code contains no $_GET / $_POST / $_REQUEST / $_SESSION
+	* [ ] Access to $_GET-parameters happens through the request-wrapper and only in constructors
+	* [ ] Direct access to $_POST parameters through the request-wrapper is only used exceptionally.
+* [ ] There are at least as many unit tests as in the last ILIAS Version and all tests pass
```

Comment:
Hmmm, this is debatable. What if a maintainers refactors the code of a component and dramatically reduces the amount of classes/methods/you name it and therefore removes tests?

Reply:
Full ack! I wanted to avoid a slow dwindling of our untit tests because of things getting removed and not replaced during refactoring and simply didn't think about this case. How about replacing this by some phpunit-code-coverage-metric? ...or if you don't think it is necessary then we simply drop it.

Reply:
Ich finde, wir können da ruhig erstmal so lassen. Weil: Es kann zwar sein, dass die weniger werden, wegen eines Refactorings. Aber ist das wirklich ein Fall, den wir *im Moment* erwarten würden? Wenn die Menge von Unit-Tests und Refactorings kommt, die das relevant machen würde, dann können wir das immer noch wieder entfernen oder verfeinern.

Reply:
Ich würde es auch so belassen. Um es aber auch wirklich einzufordern und nicht ein "Finger-in-den-Wind" halten zu ermöglichen würde ich die Zahlen vom letzten Release (Tests/Assertions) und die aktuellen zahlen direkt hier im Template einfordern. Dann sind die Reviewenden angehalten, das auch wirklich auszuführen und zu vergleichen.

Reply:
Ich würde die stelle Anpassen. Wir haben durchaus Leute, die sowas sehr wörtlich nehmen, dann schlau sind und einfach nie unit tests schreiben. Vielleicht sowas wie, "the only exception is, if tested code is completely removed due to some abandoned features or centralisation of code."

Reply:
I went with @Amstutz version here only changing the wording a little.